### PR TITLE
Security topicals

### PR DESCRIPTION
**Pull request by KillerTheFlareon**
## Short description

I'm adding red topicals and specialized red topicals to the security vending machine, for the brigmedic.

## Why we need to add this

Brigmedic was lacking anything whatsoever in their category of the security vending machine so this change adds more healing options for the brigmedic of any given security department in the same theme of "blue topicals" from the medical vending machine (and thus no longer needing to empty the medical vending machine of their blue topicals), that being red topicals (brute and ointment) and specialized topicals that excel at healing one type of brute category, but does terribly in others. 

NOTE: Until vending machines can have specific categories locked to other jobs, or someone makes a brigmedical vending machine all of security will be able to purchase these topicals due to the lack of "lock" so to speak to stop them. Some indicator saying "this is for brigmedic only" would be advisable. 

## Media (Video/Screenshots)

https://github.com/user-attachments/assets/60bb763b-52e5-486f-bfa1-861b46f5311b

## Checks

- [x] I do not require assistance to complete the PR.
- [x] Before posting/requesting review of a PR, I have verified that the changes work.
- [x] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [x] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**

:cl: KillerTheFaraon
- add: Added advanced brute themed topicals to security credit vending machine (lace, punct, bruiz) being specialized topicals.
- add: Added security general topicals, red brute and ointment.
- add: Added security medical category restricted to brigmedic only in security credit vending machine holding the new topicals.

### CODE REVIEW

File: Resources/Prototypes/_StarLight/Entities/Objects/Specific/medical.yml [hunk 1]
```diff
@@ -140,3 +140,318 @@
...
+
+- type: entity
+  name: red punct-laced brute pack
+  description: Specalized mixture of Puncturase and Security-Quality Red Goo. Best used for bullet-wounds, as it is much less effective on other types of damage.
+  parent: RedPunctPack
+  id: RedPunctPack1
+  suffix: 1
+  components:
+  - type: Sprite
+    state: puncturaseredbrutepack1
```
Comment (Rinary1) [inline Resources/Prototypes/_StarLight/Entities/Objects/Specific/medical.yml]:
```suggestion
    state: puncturaseredbrutepack
```

File: Resources/Prototypes/_StarLight/Catalog/VendingMachines/Inventories/security.yml [hunk 2]
```diff
@@ -33,3 +33,63 @@
...
+  id: RedBrutePack
+  productEntity: RedBrutePack
+  cost:
+    NTCredit: 68
+  categories:
+  - SLBrigmedic
+  conditions:
+  - !type:ListingLimitedStockCondition
+    stock: 4
+  - !type:BuyerAccessCondition
```
Comment (Rinary1) [inline Resources/Prototypes/_StarLight/Catalog/VendingMachines/Inventories/security.yml:47]:
```suggestion
  - !type:BuyerAccessCondition
    access: Brigmedic
```

File: Resources/Prototypes/_StarLight/Catalog/VendingMachines/Inventories/security.yml [hunk 3]
```diff
@@ -33,3 +33,63 @@
...
+  id: RedOintment
+  productEntity: RedOintment
+  cost:
+    NTCredit: 68
+  categories:
+  - SLBrigmedic
+  conditions:
+  - !type:ListingLimitedStockCondition
+    stock: 4
+  - !type:BuyerAccessCondition
```
Comment (Rinary1) [inline Resources/Prototypes/_StarLight/Catalog/VendingMachines/Inventories/security.yml:60]:
```suggestion
  - !type:BuyerAccessCondition
    access: Brigmedic
```

File: Resources/Prototypes/_StarLight/Catalog/VendingMachines/Inventories/security.yml [hunk 4]
```diff
@@ -33,3 +33,63 @@
...
+  id: RedPunctPack
+  productEntity: RedPunctPack
+  cost:
+    NTCredit: 102
+  categories:
+  - SLBrigmedic
+  conditions:
+  - !type:ListingLimitedStockCondition
+    stock: 4
+  - !type:BuyerAccessCondition
```
Comment (Rinary1) [inline Resources/Prototypes/_StarLight/Catalog/VendingMachines/Inventories/security.yml:73]:
```suggestion
  - !type:BuyerAccessCondition
    access: Brigmedic
```

File: Resources/Prototypes/_StarLight/Catalog/VendingMachines/Inventories/security.yml [hunk 5]
```diff
@@ -33,3 +33,63 @@
...
+  id: RedLacePack
+  productEntity: RedLacePack
+  cost:
+    NTCredit: 102
+  categories:
+  - SLBrigmedic
+  conditions:
+  - !type:ListingLimitedStockCondition
+    stock: 4
+  - !type:BuyerAccessCondition
```
Comment (Rinary1) [inline Resources/Prototypes/_StarLight/Catalog/VendingMachines/Inventories/security.yml:86]:
```suggestion
  - !type:BuyerAccessCondition
    access: Brigmedic
```

File: Resources/Prototypes/_StarLight/Catalog/VendingMachines/Inventories/security.yml [hunk 6]
```diff
@@ -33,3 +33,63 @@
...
+  id: RedBruizPack
+  productEntity: RedBruizPack
+  cost:
+    NTCredit: 102
+  categories:
+  - SLBrigmedic
+  conditions:
+  - !type:ListingLimitedStockCondition
+    stock: 4
+  - !type:BuyerAccessCondition
```
Comment (Rinary1) [inline Resources/Prototypes/_StarLight/Catalog/VendingMachines/Inventories/security.yml:99]:
```suggestion
  - !type:BuyerAccessCondition
    access: Brigmedic
```

**Review by Rinary1 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_StarLight/Entities/Objects/Specific/medical.yml [hunk 7]
```diff
@@ -140,3 +140,318 @@
...
+
+- type: entity
+  name: red lace-laced brute pack
+  description: Specalized mixture of Lacerinol and Security-Quality Red Goo. Best used for stab-wounds, as it is much less effective on other types of damage.
+  parent: RedLacePack
+  id: RedLacePack1
+  suffix: 1
+  components:
+  - type: Sprite
+    state: lacerinolbrutepack1
```
Comment (Rinary1) [inline Resources/Prototypes/_StarLight/Entities/Objects/Specific/medical.yml]:
```suggestion
    state: lacerinolbrutepack
```

**Review by Rinary1 — CHANGES_REQUESTED** [review]

**Comment by Rinary1** [comment]
changelog?

**Comment by KillerTheFlareon** [comment]
Pretty much just added your suggested changes. That's about it

**Comment by Rinary1** [comment]
> Pretty much just added your suggested changes. That's about it

lol, i mean, add changelog at pr desc, what describes what your pr adds, read changelog comment

File: Resources/Prototypes/_StarLight/Entities/Objects/Specific/medical.yml [hunk 8]
```diff
@@ -140,3 +140,318 @@
...
+
+- type: entity
+  name: red bruiz-laced brute pack
+  description: Specalized mixture of Bruizine and Security-Quality Red Goo. Best used for blunt-force trauma, as it is much less effective on other types of damage.
+  parent: RedBruizPack
+  id: RedBruizPack1
+  suffix: 1
+  components:
+  - type: Sprite
+    state: bruizinebrutepack1
```
Comment (Rinary1) [inline Resources/Prototypes/_StarLight/Entities/Objects/Specific/medical.yml]:
```suggestion
    state: bruizinebrutepack
```

**Review by Rinary1 — CHANGES_REQUESTED** [review]

**Pull request merged**